src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala

```diff
@@ -69,6 +69,16 @@ final class ScalaSensor extends Sensor {
     }
   }
 
+  def ExampleIssues: Unit = {
+    val isEmpty = List(1).size == 0
```

Comment:
INFO: List.size is O(n) ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.ListSize&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.collections.ListSize))